### PR TITLE
komodo-python-dbgp: init at 11.0

### PR DESCRIPTION
**Pull request by doronbehar**
###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by filalex77** [comment]
@GrahamcOfBorg build komodo-python-dbgp

File: pkgs/development/tools/komodo-python-dbgp/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  stdenv,
+  fetchFromGitHub,
```
Comment (filalex77) [inline pkgs/development/tools/komodo-python-dbgp/default.nix]: I believe `fetchFromGitHub` is irrelevant now.

```diff
-{
-  stdenv,
-  fetchFromGitHub,
-  pythonPackages
-}:
+{ stdenv, pythonPackages }:
```

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -17730,6 +17730,8 @@ in
 
   das_watchdog = callPackage ../tools/system/das_watchdog { };
 
+  komodo-python-dbgp = callPackage ../development/tools/komodo-python-dbgp {};
```
Comment (filalex77) [inline pkgs/top-level/all-packages.nix]: `all-packages.nix` are *not strictly* alphabetized, but could you try keeping that at least in some way?

**Comment by doronbehar** [comment]
Wow thanks for attending this @filalex77 I've forgotten I've opened this PR... I've fixed the issues. Please note there's an almost identical PR to this one for the python3 version of this package - https://github.com/NixOS/nixpkgs/pull/69198 .

**Comment by doronbehar** [comment]
Wow thanks for attending this @filalex77 I've forgotten I've opened this PR... I've fixed the issues. Please note there's an almost identical PR to this one for the python3 version of this package - https://github.com/NixOS/nixpkgs/pull/69198 .

**Review by Br1ght0ne — APPROVED** [review]

**Comment by doronbehar** [comment]
Closing as I don't see myself even using this package.

**Pull request closed without merging**